development/migrations/dependencies.rst

```diff
@@ -0,0 +1,100 @@
+============
+Dependencies
+============
+
+Dependencies tell the Migrator what order Migrations must be installed in.
+
+Declaring Dependencies
+======================
+
+In your Migration file, add a static public function named depends_on(), returning an array.
+
+.. code-block:: php
+
+    static public function depends_on()
+    {
+        return array();
+    }
+
+All you must do is add the class name of the Migration that the current Migration depends on.
+
+For example, ``phpbb\db\migration\data\v310\dev`` that 3.0.11, extensions, style update,
```

Comment:
"For example, the migration phpbb\db\migration\data\v310\dev depends on the extensions, style update, reported posts display, and timezone migrations to have already been installed."